course2_react/todo-list/src/App.js

```diff
@@ -0,0 +1,33 @@
+import logo from "./logo.svg";
+import "./App.css";
+import List from "./List";
+import { useState } from "react";
+
+function App() {
+  const [input, setInput] = useState("");
+  const [names, setNames] = useState([
+    { n1: "Harshal", id: 1 },
+    { n1: "Jayesh", id: 2 },
+    { n1: "Vicky", id: 3 },
+  ]);
+
+  const deleteName = (id) => {
+    const newName = names.filter((name) => name.id !== id);
+    setNames(newName);
```

Comment:
No need of extra constant
`setNames((names) =>names.filter((name) => name.id !== id))`